lib/ts/recipe/thirdparty/components/themes/styles.css

```diff
@@ -52,10 +51,40 @@
     justify-content: center;
     overflow: hidden;
     white-space: nowrap;
-    text-overflow: ellipsis;
     display: inline-block;
+    flex-grow: 1;
+    max-width: 100%;
 }
 
 [data-supertokens~="providerButtonText"]:only-child {
     margin: 0 auto;
 }
+
+[data-supertokens~="providerButtonText"]:not(:hover) {
+    text-overflow: ellipsis;
+}
+.scroll-text-animation:hover span,
+.scroll-text-animation:focus span {
+    display: inline-block;
+    animation-name: scroll-text;
+    animation-duration: 7s;
+    animation-timing-function: linear;
+    animation-delay: 0s;
+    animation-iteration-count: 1;
+    animation-direction: normal;
```

Comment:
Is there a way to make the animation stop (or reverse) when the text reaches the end? 
On my machine it'll scroll the provider name off the screen then go back to the start. Instead I'd prefer if it stopped once the right edge of the text is on the screen (plus some margin maybe)

Reply:
Resolved